tests/data/racket.rkt

```diff
@@ -0,0 +1,40 @@
+;;; 40 lines 14 code 15 comments 11 blanks
+#lang racket ; defines the language we are using
+
+;;; Comments
+
+;; Single line comments start with a semicolon
+
+#| Block comments
+   can span multiple lines and...
+    #|
+        they can be nested!
+    |#
+|#
+
+;; S-expression comments discard the following expression,
+;; useful to comment expressions when debugging
+#; (this expression is discarded)
```

Comment:
This line is being counted as code. Is `#` or `#;` a comment syntax? If so it is missing from the definition and is causing your file to fail the tests.

Reply:
I mentioned this in the PR body, but `#;` comments out whole s-expressions, which I don't think is something tokei can handle right now. Maybe it makes sense to just count this as code anyway because it's used to comment out blocks of code quickly, rather than to write informative comments. Should I just update the test to not count this as a comment?

Reply:
@aatxe It's up to you. You could also remove this line since I don't see how tokei could ever really count this type of comment in a language agnostic way.

Reply:
Indeed, it definitely couldn't (since this is AST aware). I'll leave it, but update the test (and add a comment that this is counted as code by tokei).